code/API_definitions/quality-on-demand.yaml

```diff
@@ -1347,6 +1351,31 @@ components:
                 status: 422
                 code: UNNECESSARY_IDENTIFIER
                 message: The device is already identified by the access token.
+    CreateSessionUnprocessableEntity422:
```

Comment:
This new schema should be merged with the codes in Generic422, but we have to keep also the other one for the retrieveSessions operation)